docs/getting-started.mdx

```diff
@@ -0,0 +1,76 @@
+---
+title: Getting Started
+nav: 0.1
+---
+Welcome to the Jotai Getting Started tutorial!
+This tutorial will introduce you to Jotai and teach you how to start using it correctly.
+
+For this tutorial, we assume that you're using Jotai with React. The example is based on [a Create-React-App folder structure](https://create-react-app.dev/docs/folder-structure) so the first thing you have to do is:
+
+`npx create-react-app my-app`
+
+## Installation
+To install the latest stable version of Jotai, run the following command:
+
+`npm install jotai` or `yarn add jotai`
+
+## Provider
+
+Provider is to provide a state for component sub tree.
+This works just like the normal React Context.
+A good place to put this is in your root component:
+
+```jsx
+import { Provider, atom, useAtom } from 'jotai'
+
+const App = () => (
+  <Provider>
+    <Input />
+    <CharCount />
+    <Uppercase />
+  </Provider>
+)
+
+export default App
+```
+
+We'll implement the components mentioned above in the following section.
+
+## atom
+
+There's an exported function called `atom` that comes with jotai, which represents a piece of state. Atoms can be read from and written to from any component.
+
+To create an atom, all you need is to pass an initial value.
+
+```jsx
+const textAtom = atom('hello')
+const textLenAtom = atom((get) => get(textAtom).length)
+const uppercaseAtom = atom((get) => get(textAtom).toUpperCase())
+```
+In the code above, beside creating sample atom we also created derived atom, which called Read-only atom pattern.
+For more detailed information about atom, We do recommend you to read [Primitives](https://jotai.org/docs/basics/primitives) section of the documentation.
+
+## useAtom
+
+The `useAtom` hook is to read an atom value in the state.
+The `useAtom` function returns the atom value and an updating function as a tuple,
+just like React's `useState`.
+
+Now that we understand main idea behand Jotai, let's create the components we mentioned above:
+
+```jsx
+const Input = () => {
+  const [text, setText] = useAtom(textAtom)
+  return <input value={text} onChange={(e) => setText(e.target.value)} />
+}
+
+const CharCount = () => {
+  const [len] = useAtom(textLenAtom)
+  return <div>Length: {len}</div>
+}
+
+const Uppercase = () => {
+  const [uppercase] = useAtom(uppercaseAtom)
+  return <div>Uppercase: {uppercase}</div>
+}
+```
```

Comment:
At last, we would want some wrapping out notes, and how we should proceed the following sections in the docs.